sunkit_image/tests/test_granule.py

```diff
@@ -0,0 +1,211 @@
+import random
+
+import numpy as np
+import pytest
+
+import sunpy
+import sunpy.map as sm
+
+import sunkit_image.granule as granule
+
+
+@pytest.fixture(scope="session")
+def inputs():
+    import warnings
+
+    with warnings.catch_warnings():
+        data_map = sunpy.map.Map("sunkit_image/tests/granule_testdata.fits")
+    test_res = 0.016
+    test_method = "li"
+    return data_map, test_res, test_method
+
+
+def test_segment(inputs):
+    """
+    Unit tests for segment() function.
+    """
+
+    data_map, test_res, test_method = inputs
+
+    # -------- positive tests -------- :

```

Comment:
Positive refers to?

Reply:
"Positive" would be a test that ensures the correct output is produced, whereas "negative" would be a test that ensures some incorrect output is not generated. This is likely unnecessary, its just something we were required to specifically include and notate for the class this was originally developed for.

Reply:
Makes sense, I would probably remove the comment now tho.